code/parse/sexp_container.cpp

```diff
@@ -109,14 +124,18 @@ list_modifier::Modifier list_modifier::get_modifier(const char *modifier_name)
 	return ListModifier::INVALID;;
 }
 
-bool list_modifier::match_name(const char *other_name) const
+const char *get_list_modifier_name(const ListModifier modifier)
```

Comment:
Should this argument be a reference?

Reply:
`ListModifier` is an alias for the `Modifier` enum class inside `list_modifier` (see sexp_container.h), so passing it by value makes sense to me. What do you think?

Reply:
I missed the fact that this was an enum.  Yes, pass-by-value is fine here.